backend/src/routes/TutorialRoute.ts

```diff
@@ -21,9 +21,9 @@ export default class TutorialRoute implements IRoute {
      */
     constructor() {
         this.router.use("/tutorial/",this.authorisationCheck);
-        this.router.get("/tutorial/", TutorialController.get); // get all? or by id?
-        this.router.post("/tutorial/", TutorialController.post); // create a tutorial
-        this.router.put("/tutorial/", TutorialController.put); // update a tutorial
+        this.router.get("/tutorial/", CMSTutorialController.get); // get all? or by id?
+        this.router.post("/tutorial/", CMSTutorialController.post); // create a tutorial
+        this.router.put("/tutorial/", CMSTutorialController.put); // update a tutorial
```

Comment:
Not sure about this one.
If I have to create a route for example admin profile then I have to create a another route file `AdminProfileRoute` and give its controller explicitly? 
Can it use the controller implicitly?

Reply:
Not sure what you mean? It goes Server -> routes -> controllers
Not server -> profile -> route -> controller
The controller are logic containers that we register depending on other factors.

Reply:
hmm.. leave it man. I dont know the structure for node js application. 
```js
Route::get('getTutorials', 'Cmscontroller@get');
```
if you create a Route class like this one. You dont have to bind those on your constructors.
```

Reply:
I can look into maybe adding decorators and binding to the decorators? I'll look into it soon if you want